react/reactHelp.md

```diff
@@ -0,0 +1,25 @@
+## How to run react prototype
+
+#### Create branch before PR(Pull Request)
+1. $ `npm install`
+2. $ `npm start`
+3. open web-browser with url `http://localhost:3000/` (can be changed)
+4. test: press arrow button (up/down/left/right)
+5. comments
+    * The bird is moving according to key input, but it's not stopping for now.
```

Comment:
are you planning on using the birds animation for the entire game? using a text based game may be easier from a development and scalable implementation standpoint?

Reply:
It was just a sample of blockly images. we can change it anytime we want. please let me know if any.